src/libraries/tests.proj

```diff
@@ -159,6 +159,104 @@
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Private.Xml/tests/XmlSchema/XmlSchemaSet/System.Xml.XmlSchemaSet.Tests.csproj" />
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Reflection.Emit.ILGeneration/tests/System.Reflection.Emit.ILGeneration.Tests.csproj" />
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Reflection.TypeExtensions/tests/System.Reflection.TypeExtensions.Tests.csproj" />
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Resources.Reader/tests/System.Resources.Reader.Tests.csproj" />
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Net.NameResolution/tests/PalTests/System.Net.NameResolution.Pal.Tests.csproj" />
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Runtime.InteropServices/tests/System.Runtime.InteropServices.Tests.csproj" />
+
+    <!-- System.InvalidOperationException - https://github.com/dotnet/runtime/issues/46759 -->
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Collections.Immutable/tests/System.Collections.Immutable.Tests.csproj" />
+
+    <!-- System.ArgumentException - https://github.com/dotnet/runtime/issues/46760 -->
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)Microsoft.Extensions.Primitives/tests/Microsoft.Extensions.Primitives.Tests.csproj" />
+
+    <!-- function signature mismatch - https://github.com/dotnet/runtime/issues/46761 -->
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.ValueTuple/tests/System.ValueTuple.Tests.csproj" />
+
+    <!-- System.Xml.Tests.VerifyException - https://github.com/dotnet/runtime/issues/46764 -->
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Private.Xml/tests/XmlSchema/XmlSchemaValidatorApi/System.Xml.XmlSchema.XmlSchemaValidatorApi.Tests.csproj" />
+
+    <!-- unreachable - https://github.com/dotnet/runtime/issues/46766 -->
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Runtime.CompilerServices.Unsafe/tests/System.Runtime.CompilerServices.Unsafe.Tests.csproj" />
+
+    <!-- https://github.com/dotnet/runtime/issues/46768 -->
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Diagnostics.DiagnosticSource/tests/TestWithConfigSwitches/System.Diagnostics.DiagnosticSource.Switches.Tests.csproj" />
+
+    <!-- https://github.com/dotnet/runtime/issues/46856 -->
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)/System.Diagnostics.FileVersionInfo/tests/System.Diagnostics.FileVersionInfo.Tests/System.Diagnostics.FileVersionInfo.Tests.csproj" />
+
+    <!-- Run only a small randomly chosen set of passing test suites on CI because running all suites is expected to take longer than a day -->
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)*\tests\**\*.Tests.csproj" />
+    <ProjectExclusions Remove="$(MSBuildThisFileDirectory)System.Buffer.Tests\tests\System.Buffer.Tests.csproj" />
```

Comment:
@steveisok Btw, the $(MSBuildThisFileDirectory)System.Buffer.Tests directory doesn't exist, so this is not testing anything.

```suggestion
<ProjectExclusions Remove="$(MSBuildThisFileDirectory)System.Buffer\tests\System.Buffer.Tests.csproj" />
```

Reply:
Thanks for the catch! There is more work being done in https://github.com/dotnet/runtime/pull/48226 that removes this line, and later the exclusions will be revisited and updated according to this tracking issue https://github.com/dotnet/runtime/issues/48079.